play-services-maps/core/hms/src/main/kotlin/org/microg/gms/maps/hms/GoogleMap.kt

```diff
@@ -309,13 +309,13 @@ class GoogleMapImpl(private val context: Context, var options: GoogleMapOptions)
     override fun setOnCameraChangeListener(listener: IOnCameraChangeListener?) = afterInitialize {
         Log.d(TAG, "setOnCameraChangeListener");
         cameraChangeListener = listener
-        it.setOnCameraIdleListener {
-            try {
-                cameraChangeListener?.onCameraChange(map?.cameraPosition?.toGms())
-            } catch (e: Exception) {
-                Log.w(TAG, e)
-            }
-        }
+//        it.setOnCameraIdleListener {
+//            try {
+//                cameraChangeListener?.onCameraChange(map?.cameraPosition?.toGms())
+//            } catch (e: Exception) {
+//                Log.w(TAG, e)
+//            }
+//        }
```

Comment:
Please don't comment those lines. If they shouldn't be here (which i assume is why you commented them), just remove them entirely.